core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -217,7 +261,12 @@ public long applyAndJournal(Supplier<JournalContext> context, NewBlockEntry entr
   public void applyAndJournal(Supplier<JournalContext> context, RenameEntry entry) {
     try {
       applyRename(entry);
-      context.get().append(JournalEntry.newBuilder().setRename(entry).build());
+      JournalEntry.Builder builder = JournalEntry.newBuilder().setRename(entry);
```

Comment:
This is in a different order than the `applyDelete` operation. Does the ordering of these matter?

Reply:
I'm pretty sure somebody made a quick hack on some issue years ago.
This part is where I'm making sure followers will see this entry with an operationId. So from their perspective, it didn't happen if it's not on the journals. 

There has to be back-off on the `DefaultFileSystemMaster` for when things go wrong for this particular master, for which we'll get here again.